utils.js

```diff
@@ -575,9 +575,9 @@ function makeDefaults(html, userID) {
 function parseAndCheckLogin(jar, defaultFuncs) {
   return function(data) {
     return bluebird.try(function() {
-      log.verbose("parseAndCheckLogin: " + data.body);
+      log.verbose("parseAndCheckLogin", data.body);
```

Comment:
This may (or may not) as well fix a memory leak I'm currently investigating, as seen here:

![](http://i.imgur.com/bhHognD.png)